Licenza utenza

izyr-collab opened the pull request:
aggiunta lincenza

izyr-collab committed:
Rinominato LICENSE in Licenza-di-walid.md

izyr-collab committed:
Aggiunta copia ufficiale della licenza

izyr-collab committed:
Modifica formattazione licenza Walid

alessiomatozzi reviewed (changes requested):
Il file della licenza va rimosso e il secondo va spostato.

izyr-collab committed:
Modifica licenza

- Rimozione LICENSE
- Spostamento Licenza-di-walid.md in Licenze-contributori/Licenza-di-walid-md

alessiomatozzi reviewed (approved):
LGTM
